addons/osfstorage/listeners.py

```diff
@@ -24,7 +24,7 @@ def checkin_files_task(node_id, user_id):
         Guid = apps.get_model('osf.Guid')
         OSFUser = apps.get_model('osf.OSFUser')
 
-        node = Guid.load_referent(node_id)
+        node, _ = Guid.load_referent(node_id)
```

Comment:
Ah, I see. This is the same fix in https://github.com/CenterForOpenScience/osf.io/pull/10844/files#diff-3c971be4babf1ba1ee7bddbee2cdbf9de7e890fd5054733476fb045fe644ae1cL27-R27. You will run into some merge conflicts after I merge this PR.